### ISSUE TITLE
Update the requirement elements related to the customer's answer for Q1(regarding discussion)

### ISSUE THREAD

**Issue by azizamankenova** [bounswe/bounswe2022group7#69]
[Requirements Draft](https://github.com/bounswe/bounswe2022group7/wiki/Requirements-(Draft)) should be reviewed and modified according to answers given in [Customer Meeting](https://github.com/bounswe/bounswe2022group7/wiki/Customer-Meeting-Report). I am responsible for checking the compatibility of requirements and making the necessary modifications related to Q1 as was decided in [Meeting #4](https://github.com/bounswe/bounswe2022group7/wiki/Meeting-Notes-%234).:
> Q1: What is the nature of the discussion forum? Will each art item have an associated discussion page?

Question 1 and its answers can be found in [Customer Meeting Report](https://github.com/bounswe/bounswe2022group7/wiki/Customer-Meeting-Report).

**Deadline: 2022-04-01, 19:59**
**Reviewer: @musasimsekdotdll**

**Comment by azizamankenova** [comment]
I have modified the discussions section in both user and system requirements in accordance with the answer of the customer and added a new feature of upvotes. Also, defined "post" and "comment" in the glossary.  
1.1.4.1, 1.1.4.2, 1.1.4.5, 1.2.2.3 were added and can be seen below.
> Glossary
**Post**: A comment written by a registered user on the discussion forum.
**Comment**: A comment written by a registered user in the comments sections of exhibitions, online galleries, collections, or pieces of art.

> 1.1.4 Discussions
1.1.4.1: Registered users and artists shall be able to publicly discuss art on the discussion forum.
1.1.4.2: Discussions shall include an upvoting system where registered users can upvote or downvote posts and comments.
1.1.4.5: Users and artists shall be able to comment and interact with other users on existing entries, which include exhibitions, online galleries, collections, or pieces of art.

> 1.2.2 Discussions
1.2.2.3: System shall display entries in discussion forum or comments according to the upvotes received. [1.1.4.2]

awaiting review from @musasimsekdotdll
